examples/messages/meta.input.json

```diff
@@ -19,7 +19,14 @@
     "id": "id-1234"
   },
   "meta": {
-    "foo": "bar"
+    "foo": "bar",
+    "workflow_tasks": {
```

Comment:
Why did this need to change?

Reply:
Good question/catch - I updated it to break for another scenario and didn't update the fixture.  It's meaningless for this test, so should be reverted.  Will fix.